examples/regression-1d/script.jl

```diff
@@ -212,19 +212,18 @@ mean(logpdf(gp_posterior(x_train, y_train, p)(x_test), y_test) for p in samples)
 # We sample 5 functions from each posterior GP given by the final 100 samples of kernel
 # parameters.
 
-plt = scatter(
-    x_train,
-    y_train;
-    xlim=(0, 1),
-    xlabel="x",
-    ylabel="y",
-    title="posterior (AdvancedHMC)",
-    label="Train Data",
-)
-for p in samples[(end - 100):end]
-    sampleplot!(plt, 0:0.02:1, gp_posterior(x_train, y_train, p); samples=5)
+plt = plot(; xlim=(0, 1), xlabel="x", ylabel="y", title="posterior (AdvancedHMC)")
+for (i, p) in enumerate(samples[(end - 100):end])
+    sampleplot!(
+        plt,
+        0:0.02:1,
+        gp_posterior(x_train, y_train, p);
+        samples=5,
+        label=(i == 1 ? "samples" : nothing),
```

Comment:
Maybe just
```suggestion
        label=(i == 1 ? "samples" : ""),
```
? I did not know that `label=nothing` is a thing :smile:

Reply:
Yeah, one of the [complaints about Plots.jl](http://juliaplots.org/MakieGallery.jl/stable/why-makie.html) is that there are plenty of different ways of saying the same thing!
I find `label = nothing` to be a bit more self-explanatory, but feel free to add&commit your suggestion if you feel moderately strongly about it:)

Reply:
No, I don't feel strongly about it. The main reason would be to avoid having two different types here but I'm fine with `nothing` as well.